backend/src/database/migrations/V1757490935__create_activityTypes_table.sql

```diff
@@ -0,0 +1,120 @@
+CREATE TABLE "activityTypes" (
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    "activityType" VARCHAR(150),
+    platform VARCHAR(150),
+    "isCodeContribution" BOOLEAN,
+    "isCollaboration" BOOLEAN,
+    "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE INDEX "idx_activityTypes_activityType"
+    ON "activityTypes" ("activityType");
```

Comment:
On this index would be good just to have id and updated at in index as this is what sequin will use to distinguish what changed... on tinybird then have sorting by activity types so the datasource is sorted by key which is filtered the most